Alert/index.html

```diff
@@ -1,18 +1,99 @@
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.3.0/font/bootstrap-icons.css"
+    />
+    <link rel="stylesheet" href="../GlobalStyles/general.css" />
+    <!-- This is reset.css -->
+    <link rel="stylesheet" href="../GlobalStyles/allstyle.css" />
+    <!-- This style is using the different component styles in other web pages -->
+    <link rel="stylesheet" href="../GlobalStyles/sitestyle.css" />
+    <link rel="stylesheet" href="style.css" />
     <title>Alert</title>
-</head>
-<body>
-    <div class ="alert alert-primary"  role="alert">Thus is a Primary Alert</div>
-    <div class ="alert alert-secondary" role="alert">Thus is a Secondary Alert</div>
-    <div class ="alert alert-success" role="alert">Thus is a success Alert</div>
-    <div class ="alert alert-danger" role="alert">Thus is a danger Alert</div>
-    <div class ="alert alert-warning" role="alert">Thus is a warning Alert</div>
-</body>
-</html>
\ No newline at end of file
+  </head>
+  <body>
+    <header>
+      <a class="logo" href ="../index.html">CUI</a>
+      <span>
+        <span><a href="#">Docs</a></span>
+        <span><a href="#">Examples</a></span>
+      </span>
+    </header>
+    <main>
+      <nav>
+        <a href="../GetStarted/index.html">Get Started</a>
+        <hr />
+        <h4>Components</h4>
+        <hr />
+        <ul>
+          <li><a href="../Avatar/index.html">Avatar</a></li>
+          <li><a href="../Alert/index.html">Alert</a></li>
+          <li><a href="../Badge/index.html">Badge</a></li>
+          <li><a href="../Button/index.html">Button</a></li>
+          <li><a href="../card/index.html">Card</a></li>
+          <li><a href="../image/index.html">Image</a></li>
+          <li><a href="../Input/index.html">Input</a></li>
+        </ul>
+      </nav>
+      <section>
+        <h3>Alert</h3>
+        <p>
+          Avatars are user profile pictures, usually found in circular shapes.
+          Avatars can be used for 'container' objects — projects, spaces,
+          repositories, etcetera — to give them a recognisable visual identity.
+        </p>
+        <hr />
+        <h4>Alert</h4>
+        <p>
+          Profile Avatars can be created using the class 'avatar', inside the
+          img tag.
+        </p>
+        <div class="container flex-container">
+          <div class="alert alert-primary">I am a Primary Alert</div>
+          <div class="alert alert-warning">I am a Warning Alert</div>
+          <div class="alert alert-success">I am a Success Alert</div>
+          <div class="alert alert-danger">I am a Danger Alert</div>
+          <div class="alert alert-secondary">I am a Secondary Alert</div>
+        </div>
+        <div
+          class="container iframe"
+          style="
+            height: 300px;
+            background-color: transparent;
+            padding-left: 0px;
+            padding-right: 0px;
+          "
```

Comment:
Suggestion: Inline styles are bad practice so your can group it in a class.

Reply:
Iframe is behaving weirdly so everywhere styling is changed I am planning to remove it 😂 But after first Submission
Thank you for your valuable feedback